pkg/cmd/agent/agent.go

```diff
@@ -181,7 +181,7 @@ func Execute(opts *AgentOpts) error {
 
 	clusterCache.SetConfigMapUpdateFunc(watchConfigFunc)
 
-	configPrefix := env.GetConfigPathWithDefault("/var/configs/")
+	configPrefix := env.GetConfigPathWithDefault(env.DefaultConfigMountPath)
```

Comment:
Now that we have `GetCostAnalyzerVolumeMountPath`, why keep using the `WithDefault` version in places like this where we know we want the mount path?

Reply:
Just trying to keep the change set minimally disruptive tbh. 1:1 string-for-const seemed perfectly unlikely to break anything. Felt that any more was scope creep -- we need to return to this question in the future. I just don't have time to revamp it all thoughtfully today.

Reply:
Really just trying to get https://github.com/kubecost/kubecost-cost-model/pull/1303 out the door!